tests/unit/synapseutils/unit_test_synapseutils_sync.py

```diff
@@ -1033,7 +1033,7 @@ def abort_side_effect(futures):
                 uploader.upload(items)
 
             # it would be aborted with Futures
-            mock_abort.assert_called_once_with([ANY])
+            mock_abort.assert_called_once()
             isinstance(mock_abort.call_args_list[0][0], Future)
```

Comment:
I know this wasn't part of this PR, but should there be an `assert` before this `isinstance`?